src/bin/generate_rr_bracket.rs

```diff
@@ -10,41 +10,15 @@ extern crate dotenv;
 // Generates a 4-player round robin bracket
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
-    #[cfg(feature = "development")]
-    {
-        create_the_bracket()?;
-    }
-    Ok(())
-}
-
-#[cfg(feature = "development")]
-fn create_the_bracket() -> anyhow::Result<()> {
-    dotenv::dotenv()?;
-    let mut db = raw_diesel_cxn_from_env()?;
-
-    run_migrations(&mut db)?;
-    let mut players = Vec::with_capacity(4);
-    players.push(
-        NewPlayer::new(
-            "foxlisk",
-            "255676979460702210",
-            Some("foxlisk#1234"),
-            Some("foxlisk"),
-        )
-        .save(&mut db)?,
-    );
-    players.push(
-        NewPlayer::new(
-            "foxlisktest",
-            "1031811909223206912",
-            Some("foxlisktest#3456"),
-            Some("bot_lisk"),
-        )
-        .save(&mut db)?,
-    );
+    dotenv::dotenv().unwrap();
```

Comment:
uhh? idk. i should maybe delete this file and figure it out properly another time